src/activities/Elsa.Activities.Timers.Hangfire/Extensions/TimersOptionsExtensions.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+
+using Elsa.Activities.Timers.Hangfire.Services;
+using Elsa.Activities.Timers.Options;
+using Elsa.Activities.Timers.Services;
+
+using Hangfire;
+
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Elsa.Activities.Timers.Hangfire.Extensions
+{
+    public static class TimersOptionsExtensions
+    {
+        /// <summary>
+        /// Add Hangfire for background processing
+        /// </summary>
+        /// <param name="timersOptions"></param>
+        public static void UseHangfireProviderCore(this TimersOptions timersOptions)
```

Comment:
Unless there's a reason to use "Core" in the name that I'm unaware of, I think we can just go with `UseHangfireProvider` or even `UseHangfire` (since it's an extension on `TimersOptions`, i.e. won't conflict with any Hangfire extensions).